xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml

```diff
@@ -18,7 +18,12 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>
+             This class provides access to the Arm64 AES Crypto intrinsics
+            
+             Arm64 CPU indicate support for this feature by setting
+             ID_AA64ISAR0_EL1.AES is 1 or better
```

Comment:
@tannergooding what does 1 or better means here? Should it be 1 or higher instead?

Reply:
These are generally singular bits and are architecture specific. They can generally only be `0` (not available) or `1` (available).

This is part of the stuff that will need to be explained in the "conceptual docs" with links (likely aka.ms links) to the ARM and x86 (Intel or AMD) specific architecture manuals.

Reply:
In this case, it looks like `ID_AA64ISAR0_EL1.AES` is 4 bits. In the latest manual it indicates:
* 0b0000 No AES instructions implemented.
* 0b0001 AESE, AESD, AESMC, and AESIMC instructions implemented.
* 0b0010 As for 0b0001, plus PMULL/PMULL2 instructions operating on 64-bit data quantities.
* All other values are reserved.

In this case, this class would be `IsSupported == true` if `AES` was `0b0001` or `0b0010`. Other values would be considered "indeterminate".